adapters/postgres/postgres.go

```diff
@@ -155,9 +155,17 @@ func OrderByRequest(r *http.Request) (string, error) {
 	return values, nil
 }
 
+// CountByRequest returns true if should use COUNT function in query
+func CountByRequest(r *http.Request) bool {
```

Comment:
you don't need a template for this you can implement like `WhereByRequest` https://github.com/nuveo/prest/blob/master/adapters/postgres/postgres.go#L44
```go
fmt.Sprintf("SELECT COUNT(%s)", val)
```